modules/improvedigitalBidAdapter.js

```diff
@@ -422,6 +409,29 @@ const ID_REQUEST = {
     }
     return null;
   },
+
+  buildSiteOrApp(request, bidderRequest) {
+    const app = {};
+    const configAppSettings = config.getConfig('app') || {};
+    const fpdAppSettings = config.getConfig('ortb2.app') || {};
+    mergeDeep(app, configAppSettings, fpdAppSettings);
+
+    if (Object.keys(app).length !== 0) {
+      request.app = app;
+    } else {
+      const site = {};
+      const url = config.getConfig('pageUrl') || deepAccess(bidderRequest, 'refererInfo.referer');
+      if (url) {
+        site['page'] = url;
+        const urlObj = parseUrl(url);
+        site['domain'] = urlObj.hostname;
```

Comment:
* site.domain
* `urlObj` is used is one place only so can be removed and replaced by `parseUrl(url)`